cl/recap/tasks.py

```diff
@@ -1538,11 +1544,15 @@ def process_recap_email(
         user_pk, settings.PACER_USERNAME, settings.PACER_PASSWORD
     )
 
+    magic_number = docket_entry["pacer_magic_num"]
     for rd in rds_created:
         fq = PacerFetchQueue.objects.create(
             user_id=user_pk, request_type=REQUEST_TYPE.PDF, recap_document=rd
         )
-        fetch_pacer_doc_by_rd(rd.pk, fq.pk)
+
+        # If we don't have a magic number avoid fetching the document
+        if magic_number:
+            fetch_pacer_doc_by_rd(rd.pk, fq.pk, magic_number)
```

Comment:
Later we'll want to send an email here, to tell the user, "Hey, you got a notification, but it didn't have a magic link, so we weren't able to get the PDF. Still, here's the contents of your alert."

Reply:
Sure, I added a TODO comment here and I'll open an issue to work on it later.